addons/project_mrp/views/mrp_production_views.xml

```diff
@@ -16,6 +16,12 @@
         <field name="model">mrp.production</field>
         <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
         <field name="arch" type="xml">
+            <field name="bom_id" position="attributes">
+                <attribute name="context">{
+                    'default_product_tmpl_id': product_tmpl_id,
```

Comment:
It is already in mrp. Because there is *no way* to use `add=` (bc the context is a string that ends with `}`), one can only copy all the keys of the context, and add what is needed inside

Reply:
Yeah makes sense, looks good to me then :100: